<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titel">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>40</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="close">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>40</y>
     <width>55</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_cut">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_clear1">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>155</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_clear2">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>152</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_sign">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>152</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_genhao">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_3">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>263</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_minus">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>263</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_0">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_dot">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_add">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_result">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="show_Wid" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>301</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="upshow">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="downshow">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_percentage">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_multiply">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>225</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_fenmu">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>193</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>231</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_6">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>225</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_9">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>187</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_devide">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>187</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_5">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>231</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_8">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>193</y>
     <width>36</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
